### PR TITLE
bug 1589950 updating no_proxy description

### PR DESCRIPTION
**Pull request by kalexand-rh**
https://bugzilla.redhat.com/show_bug.cgi?id=1589950

@sdodson, does this sound ok to you? How far back is this change valid? (The section moved to a new file in 3.10, but this section is present going back to 3.3.)

### CODE REVIEW

**Review by sdodson — COMMENTED** [review]
looks good, minor suggestions

**Comment by kalexand-rh** [comment]
Thanks @sdodson! I've made some updates based on your comments.

**Review by sdodson — APPROVED** [review]
lgtm

**Comment by kalexand-rh** [comment]
@sdodson, thanks! How far back is this change valid? (I see this section going back to 3.3.)

**Comment by sdodson** [comment]
I think it's relevant pretty far back but I don't know that it's worth backporting past 3.6.

**Comment by kalexand-rh** [comment]
3.6 it is, then. Thanks!

@openshift/team-documentation PTAL

File: install/host_preparation.adoc [hunk 1]
```diff
@@ -75,40 +75,42 @@ listed in the loop through SSH.
...
 
 [NOTE]
 ====
-This should not be confused with the global proxy values that you must set in
-your inventory file, if you are a proxy in your envirornment, which are used to
-configure specific {product-title} services with your proxy settings. See
+The `no_proxy` parameter in *_/etc/environment_* file is not the same value as
+the global proxy values that you set in your inventory file. The global proxy
+values configure specific {product-title} services with your proxy settings.
+See
```
[the remark this answers is not in the document]
Comment (mburke5678) [reply install/host_preparation.adoc]: @kalexand-rh @sdodson 
_The global proxy values configure specific {product-title} services with your proxy settings._
Which _global proxy values_ does this refer to? These or the Ansible ones?

[the remark this answers is not in the document]
Comment (mburke5678) [reply install/host_preparation.adoc]: Can we call the two sets of global proxy values something different? OpenShift Global Prxy Values and Ansible Global Proxy Values (or something more appropriate)? Seems confusing to me.

[the remark this answers is not in the document]
Comment (kalexand-rh) [reply install/host_preparation.adoc]: @mburke5678, if I change the section heading to "Setting proxy overrides," does it make more sense?

[the remark this answers is not in the document]
Comment (mburke5678) [reply install/host_preparation.adoc]: @kalexand-rh LGTM

**Pull request merged**